test/query/query.test.ts

```diff
@@ -97,3 +99,77 @@ describe('query/query', () => {
     });
   });
 });
+
+describe('2D aggregate', () => {
```

Comment:
This is calling `query()` method, why don't you put this `2D` inside `query()`.  We already talked about that you should put things under namespace -> method -> cases.  

So here is a rough sketch how you can structure it: 
```
describe("query/query", ...
  describe("query", ... // (maybe rename to `query()`) 
     describe("omitAggregatePlotWithoutDimension",     // This is the config you wanna test 
        it("?(Q) x ?(Q) should not produce MEAN(Q)xMEAN(Q) if omitAggregatePlotWithoutDimension is enabled.", ...
        it("?(Q) x ?(Q) should produce MEAN(Q)xMEAN(Q) if omitAggregatePlotWithoutDimension is disabled.", ... 
```

With this good hierarchy, you don't even need to add comment at the end.